setup.cfg

```diff
@@ -6,6 +6,7 @@ source = src
 omit =
     *test*
     *tox*
+    *examples*
```

Comment:
what is this for?

Reply:
I added the `*examples*` folder because I thought that the coverage report was including this folder. This line is now deleted.